cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -265,6 +307,62 @@ public void startStopDebugProgram(HttpRequest request, HttpResponder responder,
     startStopProgram(request, responder, namespaceId, appId, programType, runnableId, action);
   }
 
+  private void suspendResumeSchedule(HttpResponder responder, String namespaceId, String appId, String scheduleName,
+                                     String action) {
+    try {
+
+      if (!action.equals("suspend") && !action.equals("resume")) {
+        responder.sendString(HttpResponseStatus.BAD_REQUEST, "Schedule can only be suspended or resumed.");
```

Comment:
I don't have a major objection. However, in line 298, we use ``METHOD_NOT_ALLOWED``. We should standardize either way.